### PR TITLE
Merge party scale extension, update organization classifications example

### PR DESCRIPTION
**Pull request by duncandewhurst**
Related to #1179

### CODE REVIEW

**Review by odscrachel — APPROVED** [review]

File: docs/guidance/map/organization_classifications.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+Some organization classifications, such as organization scale, can be published using OCDS fields and codes. Many other classifications are context-specific, and for this case, we provide an organization classification extension that allows publishers to provide their own classifications.
 
 There are therefore two options that are encouraged for publishing organization classifications.
 
-1. For classifications that have become standardized, there are specific OCDS extension fields that ought to be used. At present, this only applies to organization scale, which ought to be disclosed using the [party scale extension](https://extensions.open-contracting.org/en/extensions/partyScale/master/). This extension adds a `scale` field to the `parties.details` block, to indicate the size or scale of an organization, in particular commercial enterprises or economic operators.
+1. For classifications that have become standardized, there are specific OCDS fields and codes that ought to be used. At present, this only applies to organization scale, which ought to be disclosed in `parties.details.scale`, using a code from the [party scale codelist](../../schema/codelists.md#party-scale).
```
Comment (jpmckinney) [inline docs/guidance/map/organization_classifications.md]:
```suggestion
1. For classifications that have become standardized, there are specific OCDS fields and codes that ought to be used. At present, this only applies to organization scale, which ought to be disclosed using the `parties.details.scale` field, using a code from the [party scale codelist](../../schema/codelists.md#party-scale).
```

File: docs/guidance/map/organization_classifications.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ A third, discouraged, example approach using local extensions is also given belo
 
 #### Organization scale
 
-In the example below, Moldova has disclosed information about the 'Companie mică' organization using the [party scale extension](https://extensions.open-contracting.org/en/extensions/partyScale/master/). The scale is given as 'micro', from the [partyScale codelist](https://extensions.open-contracting.org/en/extensions/partyScale/master/codelists/).
+In the example below, Moldova has disclosed information about the 'Companie mică' organization using `parties.details.scale`. The scale is given as 'micro', from the [partyScale codelist](../../schema/codelists.md#party-scale).
```
Comment (jpmckinney) [inline docs/guidance/map/organization_classifications.md]:
```suggestion
In the example below, Moldova has disclosed information about the 'Companie mică' organization using the `parties.details.scale` field, to which it has assigned the 'micro' code from the [partyScale codelist](../../schema/codelists.md#party-scale).
```

File: docs/schema/codelists.md [hunk 3]
```diff
@@ -333,3 +333,27 @@ The currency for each amount must be specified using the uppercase 3-letter curr
...
+
+```{versionadded} 1.2
+```
+
+The party scale codelist is used to indicate the size or scale of an organization, in particular commercial enterprises or economic operators.
+
+The codes in the codelist do not have precise definitions. Instead, they defer to local laws and regulations, for example:
+
+* [OECD: Small and Medium-Sized Enterprises (SMEs) definition](https://stats.oecd.org/glossary/detail.asp?ID=3123)
+* [European Commission: What is an SME?](https://ec.europa.eu/growth/smes/business-friendly-environment/sme-definition_en)
```
Comment (jpmckinney) [inline docs/schema/codelists.md]:
```suggestion
* [European Commission: What is an SME?](https://ec.europa.eu/growth/smes/sme-definition_en)
```

**Review by mrshll1001 — APPROVED** [review]
This is all good and guidance is clear. I think there could be some signposting in `organization_classifications.md` to indicate that the old Party Scale extension has been merged into the schema (linking to the changelog entry), for implementors that were aware of the extension before. But this is personal preference and doesn't stop the approval as the guidance here is good.

**Comment by jpmckinney** [comment]
> This is all good and guidance is clear. I think there could be some signposting in `organization_classifications.md` to indicate that the old Party Scale extension has been merged into the schema (linking to the changelog entry), for implementors that were aware of the extension before. But this is personal preference and doesn't stop the approval as the guidance here is good.

Do you mean just a "note" box at the end saying something like "This example previously used the [party scale extension] for OCDS 1.1. That extension has been merged into OCDS 1.2." ?

**Comment by jpmckinney** [comment]
@duncandewhurst @mrshll1001 See my question in my last comment.

**Comment by mrshll1001** [comment]
> Do you mean just a "note" box at the end saying something like "This example previously used the [party scale extension] for OCDS 1.1. That extension has been merged into OCDS 1.2." ?

Yes that's what I meant and I think a good way of implementing it.

**Comment by jpmckinney** [comment]
Cool - that can be added to the PR then, and then I will do a final review :)

**Comment by duncandewhurst** [comment]
@jpmckinney this is ready for your final review

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**